LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright 2017 Open Observatory of Network Interference (OONI), The Tor Project
```

Comment:
It super ugly with long lines. Can we wrap them?

Reply:
It's better to keep them on the same line in the license file to make it easier for github to parse the license file and automatically generate the license toc.

If you want to add these to headers of source code, you can wrap them.

Reply:
Truth. I was actually talking about all the following lines. You can wrap them and licensee is happy as well.